service_linux.go

```diff
@@ -260,6 +260,7 @@ func (sb *sandbox) rmLBBackend(ip, vip net.IP, fwMark uint32, ingressPorts []*Po
 	}
 }
 
+const userChain = "DOCKER-USER"
```

Comment:
Let's keep this file for service related things only.
Please move this const and the `arrangeUserFilterRule()` function to `controller.go` or `network.go`